slices/systemd.yaml

```diff
@@ -0,0 +1,635 @@
+      /usr/lib/systemd/systemd-sulogin-shell:
+      /usr/lib/systemd/systemd-sysctl:
+      /usr/lib/systemd/systemd-sysroot-fstab-check:
+      /usr/lib/systemd/systemd-sysupdate:
+      /usr/lib/systemd/systemd-sysv-install:
+      /usr/lib/systemd/systemd-time-wait-sync:
+      /usr/lib/systemd/systemd-timedated:
+      /usr/lib/systemd/systemd-tpm2-setup:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/systemd/systemd-update-done:
+      /usr/lib/systemd/systemd-update-utmp:
+      /usr/lib/systemd/systemd-user-runtime-dir:
+      /usr/lib/systemd/systemd-user-sessions:
+      /usr/lib/systemd/systemd-veritysetup:
+      /usr/lib/systemd/systemd-volatile-root:
+      /usr/lib/systemd/systemd-xdg-autostart-condition:
+
+  config:
+    contents:
+      /etc/modules-load.d/modules.conf:
+      /etc/sysctl.d/99-sysctl.conf:
+      /etc/systemd/journald.conf:
+      /etc/systemd/logind.conf:
+      /etc/systemd/networkd.conf:
+      /etc/systemd/pstore.conf:
+      /etc/systemd/sleep.conf:
+      /etc/systemd/system-generators/systemd-gpt-auto-generator:
+      /etc/systemd/system.conf:
+      /etc/systemd/user.conf:
+      /etc/xdg/systemd/user:
+      /usr/lib/environment.d/99-environment.conf:
+      /usr/lib/kernel/install.conf:
+      /usr/lib/kernel/install.d/50-depmod.install:
+      /usr/lib/kernel/install.d/55-initrd.install:
+      /usr/lib/kernel/install.d/90-loaderentry.install:
+      /usr/lib/kernel/install.d/90-uki-copy.install:
+      /usr/lib/lsb/init-functions.d/40-systemd:
+      /usr/lib/modprobe.d/systemd.conf:
+      /usr/lib/pam.d/systemd-user:
+      /usr/lib/pcrlock.d/350-action-efi-application.pcrlock:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/pcrlock.d/400-secureboot-separator.pcrlock.d/300-0x00000000.pcrlock:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/pcrlock.d/400-secureboot-separator.pcrlock.d/600-0xffffffff.pcrlock:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/pcrlock.d/500-separator.pcrlock.d/300-0x00000000.pcrlock:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/pcrlock.d/500-separator.pcrlock.d/600-0xffffffff.pcrlock:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/pcrlock.d/700-action-efi-exit-boot-services.pcrlock.d/300-present.pcrlock:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/pcrlock.d/700-action-efi-exit-boot-services.pcrlock.d/600-absent.pcrlock:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/pcrlock.d/750-enter-initrd.pcrlock:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/pcrlock.d/800-leave-initrd.pcrlock:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/pcrlock.d/850-sysinit.pcrlock:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/pcrlock.d/900-ready.pcrlock:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/pcrlock.d/950-shutdown.pcrlock:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/pcrlock.d/990-final.pcrlock:
+        arch: [amd64, arm64, riscv64]
```

Comment:
What do you think about moving these `*.pcrlock` files to a different slice?

Reply:
done